README.md

```diff
@@ -31,8 +31,8 @@ Assembler.
 Additionally, GNU libc also provides an extension, `memrchr`. This crate
 provides its own implementation of `memrchr` as well, on top of `memchr2`,
 `memchr3`, `memrchr2` and `memrchr3`. The difference between `memchr` and
-`memchr2` is that that `memchr2` permits finding all occurrences of two bytes
-instead of one. Similarly for `memchr3`.
+`memchr2` is that `memchr2` permits finding all occurrences of two bytes
+instead of one, similarly for `memchr3`.
```

Comment:
I like the rest, but I think "similarly" here should be the start of a new sentence.